src/iproto/Data/UpdateOperations/IntegerOperations.cs

```diff
@@ -0,0 +1,42 @@
+﻿namespace iproto.Data.UpdateOperations
+{
+    public class AddOperation<T> : UpdateOperation<T>
+    {
+        public AddOperation(int fieldNumber, T argument)
```

Comment:
You are leaving here a possibility for error. Tarantool is operating on Tuples, so let us work with Tuples only too. I could pass here any type, e.g. object. The whole library will go south. Factory methods with good types is not a solution.